test/extended/image_ecosystem/s2i_php.go

```diff
@@ -11,7 +11,7 @@ import (
 	exutil "github.com/openshift/origin/test/extended/util"
 )
 
-var _ = g.Describe("[sig-devex][Feature:ImageEcosystem][php][Slow] hot deploy for openshift php image", func() {
+var _ = g.Describe("[sig-examples][Feature:ImageEcosystem][php][Slow] hot deploy for openshift php image", func() {
```

Comment:
there is no such thing as `sig-examples`, please keep this devex.  I don't think "examples" would ever be a team we have.